package.json

```diff
@@ -29,10 +30,14 @@
     "haute-couture": ">=1.1.0 <2",
     "hoek": "4.x.x",
     "inert": "4.x.x",
-    "joi": "8.x.x",
+    "joi": "10.x.x",
     "labbable": "1.x.x",
+    "mysql": "^2.12.0",
+    "mysql2": "^1.1.2",
+    "objection": "^0.6.2",
     "poop": "2.x.x",
     "sails-disk": "0.10.x",
+    "sqlite3": "^3.1.8",
```

Comment:
Gotchya, yes I'll change that and just use mysql2 as main dependency, drop mysql1, and make sqlite3 a dev dependency